server/src/main/scala/drt/server/feeds/lgw/LgwForecastFeedCsvParser.scala

```diff
@@ -46,7 +46,14 @@ case class LgwForecastFeedCsvParser(fetchContent: () => Option[String]) {
               val scheduledDateAndTime = scheduledString.split(" ")
               val dateParts = scheduledDateAndTime(0).split("/")
               val timeParts = scheduledDateAndTime(1).split(":")
-              val scheduled = SDate(dateParts(2).toInt, dateParts(1).toInt, dateParts(0).toInt, timeParts(0).toInt, timeParts(1).toInt).millisSinceEpoch
+              val scheduled = SDate(
+                dateParts(2).toInt,
+                dateParts(1).toInt,
+                dateParts(0).toInt,
+                timeParts(0).toInt,
+                timeParts(1).toInt,
+                europeLondonTimeZone
```

Comment:
Would it better to alway have (as default ) europLondonTimeZone for all dates ?

Reply:
I'm not sure what you mean. Do you mean throughout the app? We store dates in UTC as then there's never any ambiguity. For this feed we're now parsing all dates as BST

Reply:
I was thinking all feeds send date with same timezone which is not case so ignore what I mentioned.